### PR TITLE
INTG-2342 Add Comment and Get Timeline  to Actions in workato

### PR DESCRIPTION
**Pull request by atexAtewologun**
Changes to process all the items in tag for actions in workato

### CODE REVIEW

File: template/template.go [hunk 1]
```diff
@@ -39,6 +39,24 @@ func (t *ServiceMethod) extractFirstTag() (string, error) {
...
+		return nil, fmt.Errorf("grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation from method %s", t.Method.Name)
+	}
+	optsTags := opts.Tags
+	var tagNames []string
+	for _, tagName := range optsTags {
+		tagNames = append(tagNames, tagName)
+	}
+
+	if len(tagNames) == 0 {
+		return nil, fmt.Errorf("couldn't find any tags for method %s", t.Method.Name)
```
Comment (sc-gv) [inline template/template.go:54]: maybe just log the error here and return an empty array?

Comment (atexAtewologun) [reply template/template.go:54]: I thought returning an empty array was a bad practice in go.

Comment (sc-gv) [reply template/template.go:54]: yeah okay return nil. dont think we need to return error

**Review by sc-gv — APPROVED** [review]
Good one @atexAtewologun

**Pull request merged**